src/JSONDB.php

```diff
@@ -235,7 +234,7 @@ public function insert($file, array $values)
         // $this->last_indexes = [ ( count( $this->content ) - 1 ) ];
         $this->commit();
 
-        // return $this->last_indexes;
+        return $this->last_indexes;
```

Comment:
To comply with return type defined in this method :+1:

Reply:
Last indexes is no longer used. It was removed on purpose. On the changelog: https://github.com/donjajo/php-jsondb/releases/tag/v3.0

Reply:
I forgot to remove the comment. I will do now.